fundingmanager.go

```diff
@@ -1697,7 +1697,7 @@ func (f *fundingManager) handleFundingCreated(fmsg *fundingCreatedMsg) {
 
 	// Inform the ChannelNotifier that the channel has entered
 	// pending open state.
-	f.cfg.NotifyPendingOpenChannelEvent(fundingOut)
+	f.cfg.NotifyPendingOpenChannelEvent(fundingOut, completeChan)
```

Comment:
so at this point the notification will trigger an update of the SCB on the gRPC API. When is the file backup triggered?

Reply:
The `chanbackup.SubSwapper` gets this notification and then updates the file backup. But you are right, I forgot the `SubscribeChannelBackups`, that should also be updated with pending channels. The synchronous RPC `ExportAllChannelBackups` always contained the unconfirmed channels because the backups were assembled on-the-fly.